queries/awk/injections.scm

```diff
@@ -1,2 +1,12 @@
-(comment) @comment
-(regex) @regex
+((comment) @injection.content
+ (#set! injection.language "comment"))
+
+((regex) @injection.content
+ (#set! injection.language "regex"))
+
+((redirected_io_statement
```

Comment:
Wait when did this appear 😆 . I have no idea of this

Reply:
No idea; cropped up when rebasing. Glitch in the git?